utility/add-new-hook.js

```diff
@@ -59,6 +59,16 @@ import { Example } from './example.stories';
 <Canvas>
   <Story story={Example} inline />
 </Canvas>
+
+## Reference
+
+\`\`\`ts
```

Comment:
What's going on here?

Reply:
Just added reference section to hook documentation template